reducer-todo/src/App.js

```diff
@@ -0,0 +1,13 @@
+import React from "react";
+import { List } from "./Components/List";
+import "./Components/Todo.css";
+
+function App() {
+  return (
+    <div className="App">
+      <List />
+    </div>
```

Comment:
This is really weird, but you're the 3rd person in a row that I'm seeing people only putting 1 component inside app.js.  Was it in the instructions somewhere and I missed it?